src/app/components/menu/menu.component.ts

```diff
@@ -0,0 +1,38 @@
+import {Component, EventEmitter, OnInit, Output} from '@angular/core';
+import {GenresHolderService} from "../../services/genres/genres-holder.service";
+import {SortService} from "../../services/sortFunctions/sort.service";
+import {ISelected} from "../../services/sortFunctions/iselected";
+import {Movie} from "../../model/movie/movie";
+
+@Component({
+  selector: 'app-menu',
+  templateUrl: './menu.component.html',
+  styleUrls: ['./menu.component.scss']
+})
+export class MenuComponent implements OnInit {
+  public genres: string[] = [];
```

Comment:
if MenuComponent is only a presentation component than it'll have genres as an input